LoopKit/GlucoseKit/CachedGlucoseObject+CoreDataClass.swift

```diff
@@ -98,6 +110,26 @@ extension CachedGlucoseObject {
             metadata: metadata
         )
     }
+
+    var trendRate: HKQuantity? {
+        get {
+            guard let trendRateUnit = trendRateUnit, let trendRateValue = trendRateValue else {
+                return nil
+            }
+            return HKQuantity(unit: HKUnit(from: trendRateUnit), doubleValue: trendRateValue.doubleValue)
+        }
+
+        set {
+            if let newValue = newValue {
+                let unit = HKUnit(from: unitString).unitDivided(by: .minute())
```

Comment:
This may throw, I think.

Reply:
This doesn't throw (and if it did, then compiler would complain about a missing `try`). However, it will crash if `unitString` isn't correct, but there isn't much we can do about that (and there is a lot of other code that does exactly this).